firstvoices/backend/search/utils/object_utils.py

```diff
@@ -171,8 +172,15 @@ def get_notes_text(dictionary_entry_instance):
 
 def get_categories_ids(dictionary_entry_instance):
     return [
-        str(id)
-        for id in dictionary_entry_instance.categories.values_list("id", flat=True)
+        str(category_id)
+        for category_id in list(
+            chain(
+                dictionary_entry_instance.categories.values_list("id", flat=True),
+                dictionary_entry_instance.categories.values_list(
+                    "parent_id", flat=True
+                ),
+            )
+        )
```

Comment:
Rather than saving the parent category id in the es index, it would probably be better to update the search api to query for all the category IDs it needs. If we story the parent category id in the index, that means we need to update potentially large amounts of dictionary entries in the index if someone edits a Category instance.

Reply:
Yes, good call. I have removed the parent category ID from the index and have checked that the functionality is now correct. 

When a child ID is given to the search API it will return only entries that have the child ID. If a parent ID is given, the API will return entries that have either the parent or a child category.